src/Common/utils.ts

```diff
@@ -165,18 +177,7 @@ class Utils {
     const hashHex = sha512.update(bytes).digest('hex').toUpperCase()
     const hash = this.toBytes(hashHex)
 
-    return hash.slice(0, 32)
-  }
-
-  /**
-   * Check if the given string is valid hex.
-   *
-   * @param input - The input to check.
-   * @returns true if the input is valid hex, otherwise false.
-   */
-  public static isHex(input: string): boolean {
-    const hexRegEx = /([0-9]|[a-f])/gimu
-    return (input.match(hexRegEx) ?? []).length === input.length
+    return hash.slice(0, hash.length / 2)
```

Comment:
Keefer suggested `hash.length / 2` over 32 to avoid using magic numbers, and to better explain what's going on here.